src/util/mod.rs

```diff
@@ -66,15 +95,34 @@ mod tests {
         // then
         let mut file = fs::File::open("temp/frame-0.txt").expect("Error opening file.");
         let mut contents = String::new();
-        file.read_to_string(&mut contents);
+        let _ = file.read_to_string(&mut contents);
         assert_eq!(contents, "3.4,6.7\n".to_owned());
 
         let mut file = fs::File::open("temp/frame-1.txt").expect("Error opening file.");
         let mut contents = String::new();
-        file.read_to_string(&mut contents);
+        let _ = file.read_to_string(&mut contents);
         assert_eq!(contents, "6.4,6.785\n".to_owned());
 
+
         // after
         fs::remove_dir_all("temp").expect("Error cleaning up test.");
     }
+
+    #[test]
+    fn body_writer() {
+        // given
+        let mut body_writer = BodyWriter::new("bodies_file");
+
+        // when
+        body_writer.write_bodies(vec![Body::new(2.1, Point::new(1.1,2.2),
+                                                Vector::new(3.4, 4.5))], "bodies".to_owned());
+//        body_writer.write_bodies(vec![Body::new(3.0, Point::new(4.0,5.0),
+//                                                Vector::zero())], "bodies".to_owned());
+
+        // then
+        let mut file = fs::File::open("bodies_file/bodies.txt").expect("Error opening file.");
+        let mut lines = String::new();
+        file.read_to_string(&mut lines);
+        assert_eq!(lines, "2.1-1.1-2.2-3.4-4.5\n".to_owned());
+    }
```

Comment:
Also don't forget to remove the directory at the end of the test, this keeps the project folder clean.